src/emcpy/plots/map_tools.py

```diff
@@ -0,0 +1,118 @@
+import cartopy.crs as ccrs
+
+
+class Domain:
+
+    def __init__(self, domain='global'):
+
+        domain = domain.lower()
+
+        map_domains = {
+            "global": self._global,
+            "conus": self._conus,
+            "north america": self._north_america,
+            "europe": self._europe
+            }
+
+        map_domains[domain]()
+
+        try:
+            return map_domains[domain]()
+        except KeyError:
+            raise TypeError(f'{domain} is not a valid domain.' +
+                            'Current domains supported are:\n' +
+                            f'{" | ".join(map_domains.keys())}"')
+
+    def _global(self):
+        """
+        Sets extent, longitude xticks, and latitude yticks
+        for a global domain.
+        """
+        self.extent = (-180, 180, -90, 90)
+        self.xticks = (-180, -120, -60, 0, 60, 120, 180)
+        self.yticks = (-90, -60, -30, 0, 30, 60, 90)
```

Comment:
one could use an user specified `dd` (domain dictionary) to specify ticks.  These can be the defaults.

Reply:
I like it! More customization for the user is a nice feature.